.aspect/workflows/deps.bzl

```diff
@@ -0,0 +1,91 @@
+"""Bazel dependencies for Aspect Workflows"""
+
+load("@bazel_tools//tools/build_defs/repo:http.bzl", _http_archive = "http_archive", _http_file = "http_file")
+load("@bazel_tools//tools/build_defs/repo:utils.bzl", "maybe")
+
+# TODO: move this to a rule set so repositories on Aspect Workflows can avoid this boilerplate
+rosetta_version = "5.9.0-rc.11"
+rosetta_integrity = {
+    "darwin_aarch64": "sha256-61vy1CrVAr4p15en1pSDFyoq5eYm1gBEN8KryRiFMxk=",
+    "darwin_x86_64": "sha256-CKdGYcNIvpWqMlo62/wqLtv0jNq9k2hWTxHlJP+7cyY=",
+    "linux_aarch64": "sha256-h4zgZ1di0kWwcBYPzFt5JErRjQPW1Zdu+DjQdKyMkeY=",
+    "linux_x86_64": "sha256-2ddpsFebV0zrj4Yl9qM4Qm37BjGm0RbeY5ofDmXEYe4",
+}
+
+# https://github.com/suzuki-shunsuke/circleci-config-merge/releases
+# https://dev.to/suzukishunsuke/splitting-circleci-config-yml-10gk
+circleci_config_merge_version = "1.1.6"
+circleci_config_merge_integrity = {
+    "darwin_aarch64": "sha256-7cQeLrSVRZR+mQu/njn+x//EIb2bhTV2+J8fafRHpr4=",
+    "darwin_x86_64": "sha256-vHKDSdDaYK58MaudJ9yOPRKh+OT/LiTQV/9E07RL8qA=",
+    "linux_aarch64": "sha256-MaXVQmRK9q9LgsfM5ZzxCIIT8rUcOBbzJ8aVDgK6zWs=",
+    "linux_x86_64": "sha256-3eYJn7dShZD1oiS3cgXfqXwdDzclf/N97A2nh7ZfW+w=",
+}
+
+def http_archive(name, **kwargs):
+    maybe(_http_archive, name = name, **kwargs)
+
+def http_file(name, **kwargs):
+    maybe(_http_file, name = name, **kwargs)
+
+# buildifier: disable=function-docstring
+def fetch_workflows_deps():
+    http_file(
```

Comment:
shouldn't this be in a `for platform, integrity in xx_integrity.keys()` ?

Reply:
yeah, I'll reduce this before landing

Reply:
✅